.travis.yml

```diff
@@ -1,17 +1,28 @@
 language: c
 
 before_install:
+    - git submodule update --init --recursive
     - sudo add-apt-repository -y ppa:snaipewastaken/ppa
     - sudo apt-get update -q
     - sudo apt-get install criterion-dev -y
+    - git clone https://github.com/redis/hiredis.git
+    - cd hiredis
+    - make
+    - sudo make install
+    - sudo ldconfig
+    - cd ..
+    - sudo redis-server&
 
 script:
     - make tests
     - ./tests/test-spellcheck
     - make
-    - ./spellcheck -v tests/test_short.txt
-    - ./spellcheck -q tests/test_short.txt -s test_save.txt
-    - ./spellcheck < tests/test_main0.txt
+    - ./spellcheck tests/test_short.txt -l
```

Comment:
How are these scripts being tested? Are y'all manually inspecting the output?

Reply:
Yeah, I check travis afterwards.

Reply:
But there're also shelltestrunner files here.